Realm.Shared/RealmList.cs

```diff
@@ -306,7 +300,7 @@ public void RemoveAt(int index)
         private void ManageObjectIfNeeded(T obj)
```

Comment:
I was only looking through public members, I'm guessing that's why this one slipped. I'll take a second look for other similar cases - I'd say rename everywhere to avoid confusion among team members as well.